### PR TITLE
add debian package search #50

### PR DESCRIPTION
**Pull request by jkahrs**

### CODE REVIEW

File: providers/debianpkg/debianpkg.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("debianpkg", &DebianPKGProvider{})
```
Comment (zquestz) [inline providers/debianpkg/debianpkg.go]: Provider struct should be just named Provider{}.

File: providers/debianpkg/debianpkg.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+// DebianPKGProvider adheres to the Provider interface.
+type DebianPKGProvider struct {
+}
+
+// BuildURI generates a search URL for WolframAlpha.
```
Comment (zquestz) [inline providers/debianpkg/debianpkg.go]: Don't think this is for WolframAlpha!

**Comment by jkahrs** [comment]
Thy for the replies @zquestz . I have updated the pr accordingly.

**Comment by KeizerDev** [comment]
@jkahrs Can you please squash this to one commit?

**Comment by zquestz** [comment]
Please squash and we will merge =)

**Comment by zquestz** [comment]
Heads up, if this doesn't get squashed soon, I am going to just add this provider myself.

**Comment by zquestz** [comment]
This has been added.

**Pull request closed without merging**